### DISCUSSION TITLE
一些二进制文件能否使用 `git lfs` 来进行管理

### DISCUSSION THREAD

**Discussion by pizeroLOL** [Class-Widgets/Class-Widgets#174] [建议]
如 7.78 MB 的 [字体文件](https://github.com/Class-Widgets/Class-Widgets/blob/main/font/HarmonyOS_Sans_SC_Bold.ttf)，0.9 MB 左右的 wav 文件，还有部分的 png 文件，这些文件的更改可能会导致仓库的体积变大，克隆的时候有点不方便。

**Comment by baiyao105** [comment]
git -f boom
